firstvoices/backend/utils/character_utils.py

```diff
@@ -108,7 +111,7 @@ def _init_custom_order(self, alphabet_length: int) -> None:
             if i not in self.exclude_chars
         ]
         if alphabet_length > len(custom_char_range):
-            raise ValueError(
+            self.logger.warning(
```

Comment:
As per discussion with Clarissa, the db now enforces a max alphabet character limit so the error in the custom sorter has been changed to a warning. This is due to the splitter also including character variants.